src/test/unit/regions/RegionTestUtilities.hpp

```diff
@@ -0,0 +1,55 @@
+/* ---------------------------------------------------------------------
+ * Numenta Platform for Intelligent Computing (NuPIC)
+ * Copyright (C) 2018, Numenta, Inc.  Unless you have an agreement
+ * with Numenta, Inc., for a separate license for this software code, the
+ * following terms and conditions apply:
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses.
+ *
+ * http://numenta.org/licenses/
+ 
+ * Author: David Keeney, July, 2018
+ * ---------------------------------------------------------------------
+ */
+ 
+/***********
+ *  Includes some routines for testing Region implementations.
+ *
+ ***********/
+#ifndef _REGIONTESTUITLITIES_HPP_
+#define _REGIONTESTUITLITIES_HPP_
+ 
+#include <iostream>
+#include <nupic/engine/Region.hpp>
+#include <nupic/algorithms/BacktrackingTMCpp.hpp>
```

Comment:
Does that have to be here?

Reply:
> #include <nupic/algorithms/BacktrackingTMCpp.hpp>

This module is shared with the BacktrackingTMRegion class and TMRegion class unit tests.  Both are coming.